src/constants.ts

```diff
@@ -140,3 +148,7 @@ export const getUsdcToken = ({
   logoURI:
     "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/logo.png",
 });
+
+export const areSettingsVisible = (env: Env): boolean => {
```

Comment:
I would not create this function here just to return `env.fiatExchangeRates.areEnabled`.

Reply:
I did it this way to centralize the Decision Point that determines if the settings will be active since this condition may change in the future and be needed in more places

Reply:
Taking into account that this is a temporary solution I would just remove this function